File: Java-Project/Task_4_Numbers.java
public class Task_4_Numbers {
    public static void main(String[] args) {
        int num1 = 20, num2=10;
        sum(num1,num2);
        difference(num1,num2);
        System.out.println("The product is "+product(num1,num2));
    }
    private static void sum(int a,int b){
        System.out.println("The sum is "+(a+b));
    }
    private static void difference(int a,int b){
        System.out.println("The difference is "+(a-b));
    }
    private static int product(int a,int b){
        return a*b;
    }
}

File: Scala-Project/Task_4_Numbers.scala
object Task_4_Numbers {
  def main(args: Array[String]): Unit = {
    val num1 = 20
    val num2 = 10

    sum(num1,num2)
    difference(num1,num2)
    println(s"The product is ${product(num1,num2)}")

  }

  val sum: (Int, Int) => Unit = (a:Int, b:Int)=> println(s"The sum is ${a+b}")
  val difference: (Int, Int) => Unit = (a:Int, b:Int)=> println(s"The difference is ${a-b}")
  val product: (Int, Int) => Int = (a:Int, b:Int) => a*b

}